go.mod

```diff
@@ -209,6 +209,6 @@ replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
 
 // If you change this please also update .github/pull_request_template.md and
 // docs/INSTALL.md.
-go 1.21.4
+go 1.22.3
```

Comment:
Why is this commit needed?

Reply:
That was just a temporary commit made in order to try to make the CI "RPC and mobile compilation check" work properly. But I've removed the commit now :)